src/MemoryStorage.php

```diff
@@ -261,53 +292,314 @@ public function __construct(Kuzzle $kuzzle)
 
     public function __call($command, $arguments)
     {
-        if (!array_key_exists($command, $this->COMMANDS)) {
+        if (!isset($this->COMMANDS[$command])) {
```

Comment:
That does not seem to be the case according to [PHP documentation](http://php.net/manual/en/function.isset.php), where it is stated that the only difference between these two methods, is that `isset` returns FALSE if an element is set but equals to NULL, where in that specific case, `array_key_exists` will return TRUE.

Other than that, using `isset` on an unexisting element of an array should simply return FALSE.